src/main/java/uk/gov/hmcts/reform/sscs/constants/WarningMessage.java

```diff
@@ -35,6 +35,7 @@
     APPOINTEE_POSTCODE(ADDRESS_POSTCODE, "Appointee postcode"),
     REPRESENTATIVE_POSTCODE(ADDRESS_POSTCODE, "Representative postcode"),
     BENEFIT_TYPE_DESCRIPTION(SscsConstants.BENEFIT_TYPE_DESCRIPTION, "Benefit type description"),
+    BENEFIT_TYPE_OTHER(SscsConstants.BENEFIT_TYPE_OTHER, "Benefit type other"),
```

Comment:
Do you need this? At least I don't think '"Benefit type other"' can ever be shown as it doesn't make sense in SSCS world which is where this error gets shown. If it is being shown, we probably need to change the wording...